components/cpm/Analysis.vue

```diff
@@ -2,50 +2,107 @@
+      </select>
     </span>
-    <div
-      class="bg-no-repeat bg-left-top bg-contain w-full h-full"
-      :style="{backgroundImage: `url(${bgImage})` }"
-    />
+    <div class="w-full h-full flex flex-wrap">
+      <div
+        class="bg-no-repeat bg-left-top bg-contain w-1/3 h-full"
+        :style="{backgroundImage: `url(${cpmImage})` }"
+      >
+        <p class="p-3 text-lg prose">
+          CPM experiment
+        </p>
+      </div>
+      <div
+        class="bg-no-repeat bg-left-top bg-contain w-1/3 h-full"
+        :style="{backgroundImage: `url(${rcmImage})` }"
+      >
+        <p class="p-3 text-lg prose">
+          RCM experiment
+        </p>
+      </div>
+      <div
+        class="bg-no-repeat bg-left-top bg-contain w-1/3 h-full"
+        :style="{backgroundImage: `url(${gcmImage})` }"
+      >
+        <p class="p-3 text-lg prose">
+          GCM experiment
+        </p>
+      </div>
+    </div>
   </div>
 </template>
 
 <script>
 export default {
+  props: {
+    domain: {
+      default: 'AL',
+      type: String
+    }
+  },
   data () {
     return {
       selectedVariable: 'pr',
-      selectedResolution: '3KM',
-      selectedCategory: 'FREQ',
-      selectedDate: '2005-05-30',
+      selectedSeason: 'DJF',
+      selectedModel: 'CLMcom-CMCC-CCLM5-0-9',
       variables: {
-        pr: 'Precipitation'
+        pr: 'Precipitation',
+        tas: 'Temperature'
       },
-      resolution: {
-        '3KM': '3KM',
-        '12KM': '12KM'
+      seasons: {
+        DJF: 'DJF',
+        JJA: 'JJA'
       },
-      categories: {
-        FREQ: 'FREQ',
-        INT: 'INT'
+      models: {
+        'CLMcom-CMCC-CCLM5-0-9': 'CLMcom-CMCC-CCLM5-0-9',
+        'CNRM-AROME41t1': 'CNRM-AROME41t1',
+        'COSMO-pompa': 'COSMO-pompa',
+        'GERICS-REMO2015': 'GERICS-REMO2015',
+        'HadREM3-RA-UM10.1': 'HadREM3-RA-UM10.1',
+        'HCLIMcom-HCLIM38-AROME': 'HCLIMcom-HCLIM38-AROME',
+        'ICTP-RegCM4-7-0': 'ICTP-RegCM4-7-0',
+        'KNMI-HCLIM38h1-AROME': 'KNMI-HCLIM38h1-AROME'
       },
-      dates: {
-        '2005-05-30': '2005-05-30',
-        '2005-08-30': '2005-08-30',
-        '2005-11-30': '2005-11-30',
-        '2006-01-01': '2006-01-01'
-      }
+      modelsList: {
+        'CLMcom-CMCC-CCLM5-0-9': { RCM: 'CCLM4-8-17_ICHEC-EC-EARTH', GCM: 'EC-EARTH' },
+        'CNRM-AROME41t1': { RCM: 'ALADIN63_CNRM-CERFACS-CNRM-CM5', GCM: 'CNRM-CM5' },
+        'COSMO-pompa': { RCM: 'CCLM4-8-17_MPI-M-MPI-ESM-LR', GCM: 'MPI-ESM-LR' },
+        'GERICS-REMO2015': { RCM: 'REMO2015_MPI-M-MPI-ESM-LR', GCM: 'MPI-ESM-LR' },
+        'HadREM3-RA-UM10.1': { RCM: 'MOHC-HadGEM3-GC3.1-N512_MOHC-HadGEM2-ES', GCM: 'HadGEM2-ES' },
+        'HCLIMcom-HCLIM38-AROME': { RCM: 'HCLIMcom-HCLIM38-ALADIN_ICHEC-EC-EARTH', GCM: 'EC-EARTH' },
+        'ICTP-RegCM4-7-0': { RCM: 'ICTP-RegCM4-7-0_MOHC-HadGEM2-ES', GCM: 'HadGEM2-ES' },
+        'KNMI-HCLIM38h1-AROME': { RCM: 'KNMI-RACMO23E_KNMI-EC-EARTH', GCM: 'EC-EARTH' }
+      },
```

Comment:
I think this data structure makes the code more difficult than it needs to be. Why is the CPM name pattern a top-level key, but rcm and GCM nested? And why are top level keys explicit strings, but 2nd level keys not?

On the page, we always show three maps. They are all "equal". What binds them together is their ancestry relationship (the model chain) and that they have been produced by a certain modelling centre. Therefore, a more faithful and useful representation would be something like:

```js
{ 
  KNMI: { 
    cpm: 'filename_pattern',
    rcm: 'filename_pattern',
    gcm: filename_pattern'
  },
  UKMO: {
  ...
  }
}
```

Reply:
Indeed I should mention this earlier. I'm not sure if we would like to display the full model/experiment name or the model/group name in the dropdown. Given that we've already displayed the name of modelling group in the leaflet map, I thought we could show more information in the dropdown to reflect the full experiment name. Therefore I took this temporary solution. Then I thought, RCM and GCM are included here for comparison only and the main goal is to showcase the CPM results. So I include RCM and GCM runs as the children of CPM...

But apparently I over-thought too much...Sorry for the confusion. Luckily it is easy to change. It is corrected now, as what you suggest.